specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -7849,6 +7989,63 @@
       },
       "description": "The language codes supported for input text by SplitSkill."
     },
+    "CustomEntityLookupSkillLanguage": {
```

Comment:
Is there a corresponding service API that a client can use to determine which languages are supported? If not, and the list is expected to be expanded over time, I would suggest considering adding one.

Reply:
There's currently no API like that. For now we have to rely on json changes like this.
Once we have such API, I think we need to migrate other skills over as well.